kitsune/sumo/jinja2/layout/breadcrumbs.html

```diff
@@ -3,7 +3,12 @@
   <ol id="breadcrumbs" class="breadcrumbs--list">
   {% for target, label in breadcrumbs %}
     {% if target %}
-      <li><a href="{{ target }}">{{ label|truncate(length=30) }}</a></li>
+      {% if target is string %}
```

Comment:
Why are you introducing this change here? Even if it is a valid change, shall we include it in a new PR since it does not seem related to the the `nofollow` directive.

Reply:
This is needed to support the new breadcrumbs format, which allows extra attributes (in this case `rel="nofollow"`) to be attached to breadcrumbs specified by sub-classes, for example, `kitsune/users/jinja2/users/edit_profile.html` below.